### DISCUSSION TITLE
Can I split the screen for multiple cameras?

### DISCUSSION THREAD

**Discussion by JSS95** [ManimCommunity/manim#3463] [Q&A]
I want to ambiently rotate a camera around an 3D object that is not on the center. Here is a brief example of my failure:

```python
from manim import *

class TestScene(ThreeDScene):
    def construct(self):
        square = Square().shift(LEFT * 3)
        cube = Cube().shift(RIGHT * 3)
        self.add_fixed_in_frame_mobjects(square)
        self.add(cube)

        self.renderer.camera.light_source.move_to(3*IN)
        self.set_camera_orientation(phi=75 * DEGREES)
        self.begin_ambient_camera_rotation(rate=90 * DEGREES)
        self.wait(4)

```

Instead of rotating the cube in-place, it satelites the object around the frame center:

https://github.com/ManimCommunity/manim/assets/52185322/90180273-f522-4b23-a691-9f947bcbbd84

I think this can be solved by having two cameras for each object, and rotating one camera around the center of the cube. Is there any way to do this? Alternative suggestion is appreciated as well!

**Comment by MrDiver** [comment]
I think the only way to do this right now is writing a custom animation for the Camera, because rotation will always be around the Origin of the scene. If you want a different point for rotation, then you would need to implement that in a custom way.

And currently we do not really have support for a multi-cam setup.
